src/app/components/partials/spinner/spinner.component.ts

```diff
@@ -1,17 +1,23 @@
 import { Component, Input, OnInit } from '@angular/core';
 import { LoaderService } from './loader.service';
+import { AnimationOptions } from 'ngx-lottie';
 
 @Component({
   selector: 'app-spinner',
+  styleUrls: [ './spinner.component.scss' ],
   template: `
-    <div class="d-flex justify-content-center">
-      <div class="spinner-grow text-center" [style.display]='visible ? "block" : "none"' role="status">
+    <div class="d-flex justify-content-center" [class.spinner]="visible">
+      <div class="text-center spinner__content" [style.display]='visible ? "block" : "none"' role="status">
```

Comment:
[style.display] - better will be changing the bootstrap helper class

Reply:
Fixed